src/components/form/PDFFunctions.js

```diff
@@ -125,14 +125,20 @@ export function getTextHeight(text,  fontSize = font.font_size.DEFAULT_FONT_SIZE
  * @param  {jsPDF}  doc           PDF document object
  * @param  {String} text          Text to be split into multiple lines
  * @param  {Number} maxLineWidth  The maximum width of the text
+ * @param  {Number} maxLines      The maximum amount of lines paragraph can hold
  * @param  {String} font          = font.font_type.DEFAULT, text font
  * @return {Array}                Array of string lines split by maxLineWidth
  */
- export function createParagraph(doc, text, maxLineWidth, fontType = font.font_type.DEFAULT, fontSize = font.font_size.MINIMUM_FONT_SIZE) {
-  //var textLines = doc.splitTextToSize(text, maxLineWidth);
-  //console.log(fontType);
-  return  doc.setFontSize(fontSize)
-             .splitTextToSize(text, maxLineWidth);
+ export function createParagraph(doc, text, maxLineWidth, maxLines = (page.PAGE_HEIGHT / font.LINE_HEIGHT),fontType = font.font_type.DEFAULT, fontSize = font.font_size.MINIMUM_FONT_SIZE) {
+   var lines = doc.setFontSize(fontSize)
+                  .splitTextToSize(text, maxLineWidth)
+
+    maxLines = lines.length - maxLines ;
+
+    if(maxLines > 0)
```

Comment:
I looked into the console.log and it was sending out '-' numbers. I added this condition to check to see if its greater than 0 and slice it. It looks similar to something I did before, just don't know how else to tackle it

Reply:
Test